tests/gp/test_hsgp_approx.py

```diff
@@ -166,25 +166,28 @@ def test_parametrization_drop_first(self, model, cov_func, X1, drop_first):
                 assert n_coeffs == n_basis, "one was dropped when it shouldn't have been"
 
     @pytest.mark.parametrize("parameterization", ["centered", "noncentered"])
-    def test_prior(self, model, cov_func, X1, parameterization):
+    @pytest.mark.parametrize("seed", list(np.arange(5) + 174))
+    def test_prior(self, model, cov_func, X1, parameterization, seed):
         """Compare HSGP prior to unapproximated GP prior, pm.gp.Latent.  Draw samples from the
         prior and compare them using MMD two sample test.  Tests both centered and non-centered
         parameterizations.
         """
+        rng = np.random.default_rng(seed)
+
         with model:
             hsgp = pm.gp.HSGP(m=[200], c=2.0, parameterization=parameterization, cov_func=cov_func)
             f1 = hsgp.prior("f1", X=X1)
 
             gp = pm.gp.Latent(cov_func=cov_func)
             f2 = gp.prior("f2", X=X1)
 
-            idata = pm.sample_prior_predictive(samples=1000)
+            idata = pm.sample_prior_predictive(samples=1000, random_seed=rng)
```

Comment:
I think its ok to decrease `samples` here to 500, will run faster which will be nice for trying the different random seeds